packages/kbn-alerts-as-data-utils/src/schemas/generated/security_schema.ts

```diff
@@ -122,6 +122,9 @@ const SecurityAlertOptional = rt.partial({
   'ecs.version': schemaString,
   'event.action': schemaString,
   'event.kind': schemaString,
+  'host.asset.criticality': schemaString,
+  'host.risk.calculated_level': schemaString,
```

Comment:
`host.risk.calculated_level` and `host.risk.calculated_score_norm` are already in the ECS component template, which is referenced by by the security alerts index. Do we need to redefine them in the security alerts component template?

Reply:
Good catch! 

I will check if I can have in alert types, but remove from field_maps, thanks!

Reply:
Tested locally, you rights, the field already in alert mappings, we probably don't need them here. Thanks!